src/rt/stdio_msvc.c

```diff
@@ -0,0 +1,154 @@
+/**
+* This module provides MS VC runtime helper function that
+* wrap differences between different versions of the MS C runtime
+*
+* Copyright: Copyright Digital Mars 2015.
+* License: Distributed under the
+*      $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0).
+*    (See accompanying file LICENSE)
+* Source:    $(DRUNTIMESRC rt/_stdio_msvc.d)
+* Authors:   Rainer Schuetze
+*/
+
+struct _iobuf
+{
+    char* _ptr;
+    int   _cnt;  // _cnt and _base exchanged for VS2015
+    char* _base;
+    int   _flag;
+    int   _file;
+    int   _charbuf;
+    int   _bufsiz;
+    char* _tmpfname;
+    // additional members in VS2015
+};
+
+typedef struct _iobuf FILE;
+extern FILE* stdin;
+extern FILE* stdout;
+extern FILE* stderr;
+
+FILE* __acrt_iob_func(int hnd);     // VS2015+
+FILE* __iob_func();                 // VS2013-
+
+int _set_output_format(int format); // VS2013-
+
+//extern const char* __acrt_iob_func;
+extern const char* _nullfunc = 0;
+
+#pragma comment(linker, "/alternatename:__acrt_iob_func=_nullfunc")
+#pragma comment(linker, "/alternatename:__iob_func=_nullfunc")
+#pragma comment(linker, "/alternatename:_set_output_format=_nullfunc")
+
+void init_msvc()
+{
+    if (&__acrt_iob_func != (void*) &_nullfunc)
+    {
+        stdin = __acrt_iob_func(0);
+        stdout = __acrt_iob_func(1);
+        stderr = __acrt_iob_func(2);
+    }
+    else if (&__iob_func != (void*) &_nullfunc)
+    {
+        FILE* fp = __iob_func();
+        stdin = fp;
+        stdout = fp + 1;
+        stderr = fp + 1;
```

Comment:
Already fixed by https://github.com/D-Programming-Language/druntime/pull/1361, i.e., in master too.